equipment_handler/test/adcs_binary_test.c

```diff
@@ -1395,6 +1400,564 @@ void binaryTest_CubeWheel3_MCU(void) {
+        printf("ADCS_set_log_config returned %d \n", test_returnState);
+        while(1);
+    }
+    
+    test_returnState = ADCS_get_log_config(&flags_arr, period, sd_card, log);
+    if (test_returnState != ADCS_OK){
+        printf("ADCS_get_log_config returned %d \n", test_returnState);
+        while(1);
+    }
+
+    // Verify Bit Mask again
+    printf("0b");
+    for (int i  = 0; i < 80; i++)
+    {
+        printf("%d", flags_arr[i]);
+
+        if ((i + 1) % 8 == 0){
+            printf(" ");
+        }
+    }
+    printf("\n");
+
+
+
+    //* Command Sequence
+    // ADCS Run Mode : State = Enabled (1)
+    printf("Running ADCS_set_enabled_state to set 1Hz loop (enabled)...\n");
+    test_returnState = ADCS_set_enabled_state(1);
+    if(test_returnState != ADCS_OK){
+        printf("ADCS_set_enabled_state returned %d \n", test_returnState);
+        while(1);
+    }
+
+
+    // Power Control : CubeControl Signal and/or Motor Power = On (1), All others = Off (0)
+    uint8_t *control = (uint8_t*)pvPortMalloc(10);
+    if (control == NULL) {
+        return ADCS_MALLOC_FAILED;
+    }
+
+    // Verify Power State(s) 
+    printf("Running ADCS_get_power_control...\n");
+    test_returnState = ADCS_get_power_control(control);
+    if(test_returnState != ADCS_OK){
+        printf("ADCS_get_power_control returned %d \n", test_returnState);
+        while(1);
+    }
+    for(int i = 0; i<10; i++){
+        printf("control[%d] = %d \n", i, control[i]);
+    }
+
+    control[Set_CubeCTRLSgn_Power] = 1;
+    control[Set_CubeCTRLMtr_Power] = 1;
+
+    test_returnState = ADCS_set_power_control(control);
+    if(test_returnState != ADCS_OK){
+        printf("ADCS_set_power_control returned %d \n", test_returnState);
+        while(1);
+    }
+    
+    // Verify Power States again
+    test_returnState = ADCS_get_power_control(control);
+    if(test_returnState != ADCS_OK){
+        printf("ADCS_get_power_control returned %d \n", test_returnState);
+        while(1);
+    }
+
+    for(int i = 0; i<10; i++){
+        printf("control[%d] = %d \n", i, control[i]);
+    }
+
+
+
+    // Set Estimation Mode : Mode = Magnetometer rate filter (2) or MEMS rate sensing (1)
+    test_returnState = ADCS_set_attitude_estimate_mode(1);
+    if(test_returnState != ADCS_OK){
+        printf("ADCS_get_power_control returned %d \n", test_returnState);
+        while(1);
+    }
+
+    adcs_state test_adcs_state;
+    printf("Running ADCS_get_current_state...\n");
+    test_returnState = ADCS_get_current_state(&test_adcs_state);
+    if(test_returnState != ADCS_OK){
+        printf("ADCS_get_current_state returned %d \n", test_returnState);
+        while(1);
+    }
+    printf("att_ctrl_mode = %d \n", test_adcs_state.att_ctrl_mode);
+    
+    // Initial Activation 1 (Detumbling, 600s)
+    printf("Initial Activation 1 : Detumbling\n");
+    printf("Running ADCS_set_attitude_ctrl_mode...\n");
+    test_returnState = ADCS_set_attitude_ctrl_mode(1, 600);
```

Comment:
Please double check before sending to the ADCS that this will send exactly the desired command.